iac-extensions/arm/src/main/java/org/sonar/iac/arm/parser/bicep/BicepGrammar.java

```diff
@@ -371,6 +373,24 @@ public UnaryOperator UNARY_OPERATOR() {
       f.unaryOperator(b.token(BicepLexicalGrammar.UNARY_OPERATOR_VALUE)));
   }
 
+  public TupleType TUPLE_TYPE() {
+    return b.<TupleType>nonterminal(BicepLexicalGrammar.TUPLE_TYPE).is(
+      f.tupleType(
+        b.token(Punctuator.LBRACKET),
+        b.zeroOrMore(
```

Comment:
I think there should always be a newline before the first element, but I can't find proper examples in docs or samples

Reply:
Yes, there should be a new line before the first element. Important is that the current implementation allows that, even if it's not implicity defined in the grammar. See test cases, the `"[typeExpr]" ` is not accepted (but it will be also not a big issue to parse it as well IMO).